xamarin/Xamarin.Forms
# [UWP] Fixed wrong font path in FontImageSourceHandler

### Description of Change ###

Fixed wrong font path in FontImageSourceHandler UWP.

_NOTE: Verified that Issue12153 (using non-embedded font ) tests continues working._

### Issues Resolved ### 

- fixes #12344 
- fixes #12635

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12344. If you can see the icon, the test has passed.

### PR Checklist ###
